dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -368,33 +368,33 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Executare ca"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "Ce cont de utilizator este folosit pentru a executa?", -1, 10, 20, 220, 20
-    CHECKBOX "Utilizator curent: %s", 14000, 10, 45, 150, 10
-    LTEXT "Protejare calculator de activiăți neautorizate", -1, 25, 57, 200, 10, WS_DISABLED
-    CHECKBOX "Această opțiune vă poate proteja calculatorul de activități care pot fi dăunătoare dar poate de asemenea duce la funcționarea incorectă a programului.", 14001, 25, 68, 200, 30, WS_DISABLED | BS_MULTILINE
-    CHECKBOX "Următorul utilizator:", 14002, 10, 100, 90, 10
+    LTEXT "Ce cont de utilizator alegeţi pentru a executa acest program?", -1, 10, 20, 220, 20
+    CHECKBOX "Utilizatorul &curent (%s)", 14000, 10, 45, 150, 10
+    LTEXT "&Protejează computerul şi datele faţă de programe neautorizate", -1, 25, 57, 200, 10, WS_DISABLED
+    CHECKBOX "Această opţiune poate preveni daunele provocate computerului şi datelor personale de către viruşi, dar selectarea acesteia poate cauza funcţionarea necorespunzătoare a programului.", 14001, 25, 68, 200, 30, WS_DISABLED | BS_MULTILINE
```

Comment:
This checkbox looks as if it would have an accelerator actually. Did you miss it?

Reply:
Yes, the checkbox has an accelerator.

![2024-03-27 00_50_31-Resource Hacker - shell32 dll mui](https://github.com/reactos/reactos/assets/113553683/13782853-6c5b-4ce5-8ce7-40e3e8e8d8c2)

Reply:
okay, I admit this one is tricky. Actually it makes more sense to have the accelerator on the checkbox, than having it on the LTEXT. but i do understand, why you do hesisate here. Let's exclude this question from this PR. You can ignore this thread, but please leave it in uncollapsed state.

Reply:
No, it doesn't seem it has an accelerator.

Reply:
It's correct what I did. I checked it.

Reply:
In xp/2k3 "&Protejează computerul şi datele faţă de programe neautorizate" and "Această opţiune poate preveni daunele provocate computerului şi datelor personale de către viruşi, dar selectarea acesteia poate cauza funcţionarea necorespunzătoare a programului." are actually the same checkbox. There is no LTEXT.